### PR TITLE
luci-app-https_dns_proxy: bugfix: subnet_addr option should be cidr

### PR DESCRIPTION
**Pull request by antoinedeschenes**
Subnet address requires CIDR notation

Signed-off-by: Antoine Deschênes <antoine@antoinedeschenes.com>

### CODE REVIEW

**Comment by feckert** [comment]
@stangri is this correct?
I am not using this app.
If ok from your side i will merge this.
Is this fix then also relevant for 19.07 and 18.06 stable branches?

**Comment by feckert** [comment]
@antoinedeschenes please also bump PKG_RELEASE number in the Makefile

File: applications/luci-app-https_dns_proxy/luasrc/model/cbi/https_dns_proxy.lua [hunk 1]
```diff
@@ -97,7 +97,7 @@ lp.write = function(self, section, value)
 end
 
 sa = s3:option(Value, "subnet_addr", translate("Subnet address"))
-sa.datatype = "ip4prefix"
+sa.datatype = "cidr"
```
Comment (stangri) [inline applications/luci-app-https_dns_proxy/luasrc/model/cbi/https_dns_proxy.lua]: I don't think ```cidr``` is a valid data-type. On 18.06.4 it produces:
```
Failed to execute cbi dispatcher target for entry '/admin/services/https_dns_proxy'.
The called action terminated with an exception:
/usr/lib/lua/luci/cbi.lua:165: Datatype error, bad token "cidr"
stack traceback:
	[C]: in function 'error'
	/usr/lib/lua/luci/cbi.lua:165: in function 'compile_datatype'
	/usr/lib/lua/luci/cbi.lua:189: in function 'verify_datatype'
	/usr/lib/lua/luci/cbi.lua:1497: in function 'validate'
	/usr/lib/lua/luci/cbi.lua:1413: in function 'parse'
	/usr/lib/lua/luci/cbi.lua:1549: in function 'parse'
	/usr/lib/lua/luci/cbi.lua:248: in function 'parse'
	/usr/lib/lua/luci/cbi.lua:1184: in function 'parse'
	/usr/lib/lua/luci/cbi.lua:248: in function 'parse'
	/usr/lib/lua/luci/cbi.lua:387: in function 'parse'
	/usr/lib/lua/luci/dispatcher.lua:856: in function </usr/lib/lua/luci/dispatcher.lua:835>
```

I suggest ```host```.

**Comment by feckert** [comment]
Thanks for the pullrequest and the fix but this is superseded by https://github.com/openwrt/luci/pull/3152

**Pull request closed without merging**